### ISSUE TITLE
Successor for Bacon.fromBinder

### ISSUE THREAD

**Issue by raimohanska** [baconjs/bacon.js#704]
In #703 it was suggested that this version of `Bacon.fromEvent` should be added:

```js
Bacon.fromEvent(
    (listener) => window.addEventListener('scroll', listener, {passive: true}),
    (listener) => window.removeEventListener('scroll', listener, {passive: true}),
    () => window.scrollX
);
```

It has the virtue of being more readable than `Bacon.fromBinder`. From my point of view, it also looks like it should be returning a `Property` because the third function argument is used to get a current value. The initial value of the `Property` could be set at the time of creation by calling the third function.

**Comment by semmel** [comment]
Ehm isn't that pull request about
```javascript
Bacon.fromEvent(window, 'scroll', { passive: true }, event => event.prop)
``` 
I mean about `Bacon.fromEvent` and not  `Bacon.fromBinder`? 
I find `Bacon.fromBinder` as general adapter just find the way it is.

**Comment by raimohanska** [comment]
Ysh

**Comment by raimohanska** [comment]
Oops sorry

**Comment by raimohanska** [comment]
Yeah there was this suggestion in the comments

**Comment by steve-taylor** [comment]
Given the symmetry of bind/unbind calls, how about something like this to avoid duplication:

```js
Bacon.fromEventBinder(
    window,
    (binder) => binder('scroll', listener, {passive: true}),
    () => window.scrollX
);
```

**Comment by steve-taylor** [comment]
> I find `Bacon.fromBinder` as general adapter just fine the way it is.

Agreed. There will always be use cases for `Bacon.fromBinder` in its current form that don't involve `EventTarget`s or `EventEmitter`s.

**Comment by steve-taylor** [comment]
I'll create a new PR shortly for the following:

```ts
Bacon.fromEvent(
    target: EventTarget|EventEmitter,
    eventSource: string|Function,
    eventTransformer?: Function
)
```

It will be an alternative to #703.

**Comment by steve-taylor** [comment]
See #710 and [related documentation](https://github.com/steve-taylor/bacon.js/blob/from-event-custom-binder/README.md#bacon-fromevent)

**Comment by steve-taylor** [comment]
Might as well close this now that #710 is merged.
